src/locales/en/translation.json

```diff
@@ -608,7 +609,9 @@
     "commissionBounds": "Commission Bounds",
     "commissionRates": "Commission Rates",
     "cumulativeVoting": "Cumulative Voting %",
+    "debondingEnd": "End of debonding",
     "delegators": "Delegators",
+    "emptyDebondingList": "There are no debonding delegations to an account.",
```

Comment:
On validator view I would say: No one is debonding delegations from this validator